UefiPayloadPkg/UefiPayloadEntry/FitUniversalPayloadEntry.c

```diff
@@ -409,21 +311,25 @@ BuildFitLoadablesFvHob (
 
   Status = FdtCheckHeader (Fdt);

```

Comment:
Copy feedbacks from Dhaval Sharma:
BuildFitLoadablesFvHob:
•	Fdt variable is not initialized.
•	It ONLY gets initialized if GuidHob is found. What if it is not found?
•	FdtCheckHeader still evaluating it?

I think we should initialize 'Fdt' pointer to NULL from the beginning and return EFI_UNSUPPORTED when either (Fdt == NULL) || (FdtCheckHeader(Fdt))